git-extra/astextplain

```diff
@@ -8,6 +8,9 @@ fi
 
 # XXX output encoding (UTF-8) hardcoded
 case "$1" in
+	*.ods | *.ODS | *odf |*.ODF | *odt | *.ODT)
+		odf2txt "$1" || cat "$1"
```

Comment:
The program `odf2txt` does not exist in Git for Windows' SDK. It will need to be provided in a Pacman package before this PR can be merged.